dash/dash.py

```diff
@@ -912,7 +923,85 @@ def dispatch(self):
                 c['id'] == component_registration['id']
             ][0])
 
-        return self.callback_map[target_id]['callback'](*args)
+        output_value = self.callback_map[target_id]['func'](*args)
+
+        # Only validate if we get required information from renderer
+        # and validation is not turned off by user
+        if (
+                not self.config.suppress_validation_exceptions and
+                'namespace' in output and
+                'type' in output
+        ):
```

Comment:
This would fit without the ()
```
if not self.config.suppress_validation_exceptions and \
                'namespace' in output and 'type' in output:
```